app/build.gradle.kts

```
@@ -20,11 +20,17 @@ android {
     signingConfigs {
-        create("release") {
-            storeFile = file(providers.gradleProperty("LAZYPIZZA_STORE_FILE").get())
-            storePassword = providers.gradleProperty("LAZYPIZZA_STORE_PASSWORD").get()
-            keyAlias = providers.gradleProperty("LAZYPIZZA_KEY_ALIAS").get()
-            keyPassword = providers.gradleProperty("LAZYPIZZA_KEY_PASSWORD").get()
+        register("release") {
+            val storeFilePath = providers.gradleProperty("LAZYPIZZA_STORE_FILE").orNull
+            val storePass = providers.gradleProperty("LAZYPIZZA_STORE_PASSWORD").orNull
+            val keyAliasVal = providers.gradleProperty("LAZYPIZZA_KEY_ALIAS").orNull
+            val keyPass = providers.gradleProperty("LAZYPIZZA_KEY_PASSWORD").orNull
+            if (storeFilePath != null && storePass != null && keyAliasVal != null && keyPass != null) {
```

detekt (1.23.8)

[warning] This condition is too complex (4). Defined complexity threshold for conditions is set to '4'

(detekt.complexity.ComplexCondition)